accname/name/comp_host_language_label.tentative.html

```diff
@@ -0,0 +1,86 @@
+    <script src="/wai-aria/scripts/aria-utils.js"></script>
+  </head>
+  <body>
+    <!-- Ambiguity coming from Host Language Label in accname specs (https://www.w3.org/TR/accname-1.2/#comp_host_language_label) - PR that tries to solve it https://github.com/w3c/aria/pull/2405 -->
+    <h2>HTML input with label with role presentation</h2>
+    <h3>HTML input label/for</h3>
+    <label role="presentation" for="lrp">input label</label>
+    <input id="lrp" type="text" data-expectedlabel="input label" data-testname="html: label[for][role presentation]" class="ex">
+    <h3>HTML label encapsulation</h3>
+    <label role="presentation">
+      <span>input label</span>
+      <input type="text" data-expectedlabel="input label" data-testname="html: label[role presentation] encapsulation" class="ex">
+    </label>
+
+    <h2>HTML input with role="presentation" and label associated (presentational roles conflict resolution)</h2>
+    <h3>HTML input label/for</h3>
+    <label for="irn">input label</label>
+    <input id="irn" type="text" data-expectedlabel="input label" data-testname="html: label[for]input[role presentation]" class="ex" role="presentation">
+    <h3>HTML label encapsulation</h3>
+    <label>
+      <span>input label</span>
+      <input type="text" data-expectedlabel="input label" data-testname="html: input[role presentation] with label encapsulation" class="ex" role="presentation">
+    </label>
+    
+    <h2>HTML input with role="presentation" and label with role presentation (presentational roles conflict resolution)</h2>
+    <h3>HTML input label/for</h3>
+    <label role="presentation" for="lirp">input label</label>
+    <input id="lirp" type="text" data-expectedlabel="input label" data-testname="html: label[for][role presentation] input[role presentation]" class="ex" role="presentation">
+    <h3>HTML label encapsulation</h3>
+    <label role="presentation">
+      <span>input label</span>
+      <input type="text" data-expectedlabel="input label" data-testname="html: input[role presentation] with label[role presentation] encapsulation" class="ex" role="presentation">
+    </label>
+
+    <h2>HTML input with role="presentation" and disabled with label associated</h2>
+    <h3>HTML input label/for</h3>
+    <label for="irpd">input label</label>
+    <input id="irpd" type="text" data-expectedlabel="" data-testname="html: label[for]input[role presentation][disabled]" class="ex" role="presentation" disabled>
+    <h3>HTML label encapsulation</h3>
+    <label>
+      <span>input label</span>
+      <input type="text" data-expectedlabel="" data-testname="html: input[role presentation][disabled] with label encapsulation" class="ex" role="presentation" disabled>
+    </label>
+
+    <h2>HTML img with role="presentation" and non empty alt</h2>
+    <img src="https://www.w3.org/assets/logos/w3c/w3c-no-bars.svg" role="presentation" alt="w3c logo" data-expectedlabel="" data-testname="html: img[role presentation][non-empty alt]">
```

Comment:
You're missing class="ex" on this one and some or all below. 

Since these files are new, better to change to the new convention: "ex-label" rather than the ambiguous "ex" I started with.

Reply:
For future reference, when you forget, you'll notice your results list being a lot shorter than your test list.